docs/manual-EN/5.appendix/sql-ngql.md

```diff
@@ -0,0 +1,194 @@
+
+### Data Query Language (DQL)
+
+DML statements are used for performing queries on the data. This section compares the how SQL and nGQL query data.
+
+```sql
+SELECT
+ [DISTINCT]
+ select_expr [, select_expr] ...
+ [FROM table_references]
+ [WHERE where_condition]
+ [GROUP BY {col_name | expr | position}]
+ [HAVING  where_condition]
+ [ORDER BY {col_name | expr | position} [ASC | DESC]]
+```
+
+```SQL
+GO [[<M> TO] <N> STEPS ] FROM <node_list>
+ OVER <edge_type_list> [REVERSELY] [BIDIRECT]
+ [WHERE where_condition]
+ [YIELD [DISTINCT] <return_list>]
+ [| ORDER BY <expression> [ASC | DESC]]
+ [| LIMIT [<offset_value>,] <number_rows>]
+ [| GROUP BY {col_name | expr | position} YIELD <col_name>]
+
+<node_list>
+   | <vid> [, <vid> ...]
+   | $-.id
+
+<edge_type_list>
+   edge_type [, edge_type ...]
+
+<return_list>
+    <col_name> [AS <col_alias>] [, <col_name> [AS <col_alias>] ...]
+```
+
+### Data Control Language (DCL)
+
+DCL includes commands such as `GRANT` and `REVOKE` which mainly deals with the rights, permissions and other controls of the database system.
+
+Name                    | SQL                   | nGQL
+-------------------------| ------------------------ | -----------
+Create user | CREATE USER | CREATE USER
+Drop user | DROP USER | DROP USER
+Change password | SET PASSWORD | CHANGE PASSWORD
+Grant privileges | GRANT `<priv_type>` ON [object_type] TO `<user>`| GRANT ROLE `<role_type>` ON `<space>` TO `<user>`
+Revoke privileges | REVOKE `<priv_type>` ON [object_type] TO `<user>` | REVOKE ROLE `<role_type>` ON `<space>` FROM `<user>`
+
+## CRUD
+
+This section gives some query examples for your reference.
+
+### Inserting Data
+
+```sql
+mysql> INSERT INTO player VALUES (100, 'Tim Duncan', 42);
+
+nebula> INSERT VERTEX player(name, age) VALUES 100: ('Tim Duncan', 42);
+```
+
+### Querying Data
+
+Find the player in the `player` table whose id is 100 and output the `name` property in RDBMS:
+
+```sql
+mysql> SELECT player.name FROM player WHERE player.id = 100;
+
+nebula> FETCH PROP ON player 100 YIELD player.name;
+```
+
+### Updating Data
+
+```sql
+mysql> UPDATE player SET name = 'Tim';
+
+nebula> UPDATE VERTEX 100 SET player.name = "Tim";
+```
+
+### Deleting Data
+
+```sql
+mysql> DELETE FROM player WHERE name = 'Tim';
+
+nebula> DELETE VERTEX 121;
+nebula> DELETE EDGE follow 100 -> 200;
+```
+
+## Sample Queries
+
+### Query 1
+
+Find players who are younger than 36.
+
+```sql
+mysql> SELECT player.name
+FROM player
+WHERE player.age < 36;
+```
+
+The query in nGQL is a bit different because we need to create an index before filtering a property. See [Index Doc](https://docs.nebula-graph.io/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/) for more information.
+
+```ngql
+nebula> CREATE TAG INDEX player_age ON player(age);
+nebula> REBUILD TAG INDEX player_age OFFLINE;
+nebula> LOOKUP ON player WHERE player.age < 36;
+```
+
+### Query 2
+
+List Tim Duncan and the team he served.
```

Comment:
suggestion: "Find Tim Duncan and list all the teams that he served or is serving"?

[always include the linking words for a clause to make the sentence more understandable".